distributed/scheduler.py

```diff
@@ -4999,18 +4999,12 @@ async def remove_worker(self, comm=None, address=None, safe=False, close=True):
 
             self.remove_resources(address)
 
-            dh: dict = parent._host_info.get(host)
-            if dh is None:
-                parent._host_info[host] = dh = {}
```

Comment:
this never happens, because if it did everything would fall over on ``dh["addresses"]`` on the next row